integration/tpl/utils.libsonnet

```diff
@@ -96,8 +96,8 @@ local ParseURL(rawURL) =
 local ComposeService(name, definition, additionalAliases=[]) =
   {
     [name]: definition {
-      networks: {
-        main: {
+      networks+: {
```

Comment:
I see it inherits from the other object. TIL.

Reply:
Exactly, I wanted to be able to combine the `ipv4_address` from the backends/httpdetails.libsonnet file with the aliases defined here. I wasn't sure if this would work with the nested keys (`main` inside of `networks`) but it looks like it does.